tests/test-notice-visible-correct-role.php

```diff
@@ -71,7 +72,8 @@ public function test_user_can_see_notice() {
 	 * Test that if capability is not passed, the notice is visible only to the user's with `manage_options` cap.
 	 */
 	public function test_user_can_see_notice_without_capability() {
-		$astra_notices = Astra_Notices::get_instance();
+		// Singleton is not used here as that causes the notices registered in one test to be visible in the next.
+		$astra_notices = new Astra_Notices();
```

Comment:
Are we removing singleton from the plugin also?

Reply:
Nope, just from tests

Reply:
Alright.